controller/config.go

```diff
@@ -165,12 +165,12 @@ const (
 	// which should be more than enough time for a debugging session.
 	MaxDebugLogDuration = "max-debug-log-duration"
 
-	// AgentLogfileMaxSize is the maximum file size in MB of each
-	// agent/controller log file.
+	// AgentLogfileMaxSize is the maximum file size in MB of each agent log
```

Comment:
For consistency with the previous example, I'd move the unit to the end:

> AgentLogfileMaxSize is the maximum file size of each agent log, in MB.

Reply:
Fixed.